packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -2432,6 +2432,25 @@ void main() {
     final TextField textField = tester.widget(find.byType(TextField));
     expect(textField.keyboardType, TextInputType.text);
   });
+
+  testWidgets('DropdownMenu with an alignmentOffset does not throw errors', (WidgetTester tester) async {
```

Comment:
I do not think that this test is testing the right thing? The `MenuAnchor` widget accepts a nullable alignmentOffset. So it would not "crash" if you gave it an alignment offset.

Instead, grab the `MenuAnchor` from the subtree and compare the alignmentOffset in an expect:

```
final MenuAnchor menuAnchor = tester.widget<MenuAnchor>(find.byType(MenuAnchor));

expect(menuAnchor.alignmentOffset, alignmentOffset);
```

Reply:
To expand on this a bit: This PR adds the `alignmentOffset` parameter to `DropdownMenu`, when previously it could only be configured for `MenuAnchor`. So if the changes to dropdown_menu.dart were reverted, attempting to set an offset this way would throw an exception.

With that in mind, I think the new version of this test is much better than what I'd suggested beforehand 👍